neard/src/cli.rs

```diff
@@ -250,6 +250,9 @@ impl RunCmd {
         let sys = actix::System::new();
         sys.block_on(async move {
             nearcore::start_with_config(home_dir, near_config);
+            tokio::signal::ctrl_c().await.unwrap();
+            info!("Got Ctrl+C, stopping");
+            actix::System::current().stop();
```

Comment:
do we have some existing ctrlc handling somewhere else? I *think* I saw related hacks somewhere else...

Reply:
Only in tests as far as I can tell. Specifically `test-utils/actix-test-utils/src/lib.rs` and `pytest/lib/cluster.py`. I am looking into removing those. Cc: @miraclx

Reply:
I’ve looked at `actix-test-utils` and it could be because my async-fu is insufficient but I couldn't figure out how to simplify code there. Since it’s test it’s probably fine even if the additional thread isn’t actually necessary.